library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -86,7 +86,6 @@ libspdm_get_spdm_response_func libspdm_get_response_func_via_request_code(uint8_
         #endif /* LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP */
     };
 
-    LIBSPDM_ASSERT(request_code != SPDM_RESPOND_IF_READY);
```

Comment:
How can this be? How are the unit tests passing?

Reply:
No idea on why this is written in this.
Responder unit test does not cover this function, because the main purpose is to test the requester message parsing, but not the state machine.


I tested the IF_READY function by setting one time "spdm_context->response_state = LIBSPDM_RESPONSE_STATE_NOT_READY" in responder.

The SPDM_RESPOND_IF_READY is issued from requester and the sequence can continue.
So I believe the rest flow works.

Reply:
Ok I filed https://github.com/DMTF/spdm-emu/issues/234 against spdm-emu to provide some coverage.